From db0d7c564822a0d0d28fa2a52fe613e297e80bef Mon Sep 17 00:00:00 2001
From: Antoine Martin <dev@ayakael.net>
Date: Thu, 16 Dec 2021 02:38:47 +0000
Subject: [PATCH 1/1] Force MicrosoftNetCoreIlasmPackageRuntimeId as

Runtime doesn't know to look for linux-musl version of
Microsoft.NETCore.Ilasm nupkgs provided by stage0 and dotnet
build. This forces looking for it

---
 Directory.Build.props | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Directory.Build.props b/Directory.Build.props
index 5cb87bb7c25..24efb6ba2b3 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -168,8 +168,7 @@
     <_toolRuntimeRID Condition="'$(_runtimeOS)' == 'maccatalyst' or '$(_runtimeOS)' == 'ios' or '$(_runtimeOS)' == 'iOSSimulator' or '$(_runtimeOS)' == 'tvos' or '$(_runtimeOS)' == 'tvOSSimulator'">osx-x64</_toolRuntimeRID>
 
     <!-- There are no non-portable builds for Ilasm/Ildasm -->
-    <MicrosoftNetCoreIlasmPackageRuntimeId Condition="'$(PortableBuild)' != 'true' and '$(_portableOS)' == 'linux'">linux-$(_hostArch)</MicrosoftNetCoreIlasmPackageRuntimeId>
-    <MicrosoftNetCoreIlasmPackageRuntimeId Condition="'$(MicrosoftNetCoreIlasmPackageRuntimeId)' == ''">$(_toolRuntimeRID)</MicrosoftNetCoreIlasmPackageRuntimeId>
+    <MicrosoftNetCoreIlasmPackageRuntimeId>linux-musl-$(_hostArch)</MicrosoftNetCoreIlasmPackageRuntimeId>
 
     <PackageRID>$(_packageOS)-$(TargetArchitecture)</PackageRID>
 
-- 
2.34.1

